package.json

```diff
@@ -23,6 +23,8 @@
     "autoprefixer-loader": "^3.2.0",
     "babel-core": "^6.26.3",
     "babel-loader": "^8.2.2",
+    "babel-plugin-polyfill-corejs3": "^0.2.0",
```

Comment:
Any reason those were added?

Reply:
I can't remember eventualy. And, I'm failing to properly remove it (see #1549)

Reply:
Hmmmm. I did:

1. rm -rf  node_modules/babel-plugin-polyfill-corejs3
2. rm package-lock.json
3. remove the babel-plugin-polyfill-corejs3 line in package.json

And, I get this when runing make:

```
ERROR in ./src/files.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: [BABEL]: Cannot find module 'babel-plugin-polyfill-corejs3'
Require stack:
- /var/www/nextcloud/apps/groupfolders/node_modules/@babel/preset-env/lib/index.js
- /var/www/nextcloud/apps/groupfolders/node_modules/@babel/core/lib/config/files/module-types.js
- /var/www/nextcloud/apps/groupfolders/node_modules/@babel/core/lib/config/files/configuration.js
- /var/www/nextcloud/apps/groupfolders/node_modules/@babel/core/lib/config/files/index.js
- /var/www/nextcloud/apps/groupfolders/node_modules/@babel/core/lib/index.js
- /var/www/nextcloud/apps/groupfolders/node_modules/babel-loader/lib/index.js
- /var/www/nextcloud/apps/groupfolders/node_modules/loader-runner/lib/loadLoader.js
- /var/www/nextcloud/apps/groupfolders/node_modules/loader-runner/lib/LoaderRunner.js
- /var/www/nextcloud/apps/groupfolders/node_modules/webpack/lib/NormalModule.js
- /var/www/nextcloud/apps/groupfolders/node_modules/webpack/lib/NormalModuleFactory.js
- /var/www/nextcloud/apps/groupfolders/node_modules/webpack/lib/Compiler.js
- /var/www/nextcloud/apps/groupfolders/node_modules/webpack/lib/webpack.js
- /var/www/nextcloud/apps/groupfolders/node_modules/webpack-cli/lib/webpack-cli.js
- /var/www/nextcloud/apps/groupfolders/node_modules/webpack-cli/lib/bootstrap.js
- /var/www/nextcloud/apps/groupfolders/node_modules/webpack-cli/bin/cli.js
- /var/www/nextcloud/apps/groupfolders/node_modules/webpack/bin/webpack.js (While processing: /var/www/nextcloud/apps/groupfolders/node_modules/@babel/preset-env/lib/index.js)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
    at Function.Module._load (internal/modules/cjs/loader.js:667:27)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (/var/www/nextcloud/apps/groupfolders/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/var/www/nextcloud/apps/groupfolders/node_modules/@babel/preset-env/lib/index.js:33:35)
    at Module._compile (/var/www/nextcloud/apps/groupfolders/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (/var/www/nextcloud/apps/groupfolders/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at loadCjsDefault (/var/www/nextcloud/apps/groupfolders/node_modules/@babel/core/lib/config/files/module-types.js:77:18)
    at loadCjsOrMjsDefault (/var/www/nextcloud/apps/groupfolders/node_modules/@babel/core/lib/config/files/module-types.js:49:16)
    at loadCjsOrMjsDefault.next (<anonymous>)
    at requireModule (/var/www/nextcloud/apps/groupfolders/node_modules/@babel/core/lib/config/files/plugins.js:184:44)
    at requireModule.next (<anonymous>)
 @ multi ./src/files.js files[0]
```

Reply:
Ok interesting, then I' fine with keeping that for now. I'll need to look into migrating the webpack config of groupfolders to our shared config anyways, which will then better handle all the dev dependencies through the shared config.

Reply:
Should be fine once rebasing to master with https://github.com/nextcloud/groupfolders/pull/1560 included. Then please just remove the changes from the package* files.

Reply:
I've tried rebasing my branch to latest master and now I'm getting this:

```
cyrille@vm-cyrille-debian:/var/www/nextcloud/apps/groupfolders$ npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: undefined@undefined
npm ERR! Found: eslint-plugin-promise@5.1.0
npm ERR! node_modules/eslint-plugin-promise
npm ERR!   peer eslint-plugin-promise@"^4.2.1 || ^5.0.0" from eslint-config-standard@16.0.3
npm ERR!   node_modules/eslint-config-standard
npm ERR!     peer eslint-config-standard@"^16.0.1" from @nextcloud/eslint-config@5.1.0
npm ERR!     node_modules/@nextcloud/eslint-config
npm ERR!       dev @nextcloud/eslint-config@"^5.1.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer eslint-plugin-promise@"^4.2.1" from @nextcloud/eslint-config@5.1.0
npm ERR! node_modules/@nextcloud/eslint-config
npm ERR!   dev @nextcloud/eslint-config@"^5.1.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /home/cyrille/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/cyrille/.npm/_logs/2021-06-22T14_19_21_721Z-debug.log
cyrille@vm-cyrille-debian:/var/www/nextcloud/apps/groupfolders$
```

I don't really understand the problem. It doesn't seems to be a problem with `eslint-plugin-promise` eventualy. The line `npm ERR! While resolving: undefined@undefined` is really weird

Reply:
Do you want me to force-push my changes now?

Reply:
@StCyr : I don't see the `eslint-plugin-promise` dependencie in your `package.json`. Maybe it is it ?

Same thing for the `eslint-config-standard` package.

Reply:
Also, maybe it's because I'm using npm 7. I've already had quite a few issue with it and package-lock.json

Reply:
> Also, maybe it's because I'm using npm 7. I've already had quite a few issues with it and package-lock.json

Shouldn't be an issue as i also have that here.

Could you try to run `npm ci` instead of `npm i` so that it just takes the currently installed packages from the lockfile without updating it?

Reply:
Thanks for the help @juliushaertl,

Eventualy, I've restored files package.json and package-lock.json from the master branch and my branch now build properly.